pkg/getter/getter.go

```diff
@@ -27,6 +27,8 @@ import (
 type Getter interface {
 	//Get file content by url string
 	Get(url string) (*bytes.Buffer, error)
+	//Get file content by url, username and password strings
+	GetWithCredentials(href, username, password string) (*bytes.Buffer, error)
```

Comment:
Let's remove this. A Getter should just implement `.Get()` with a URL. If a username/password is provided as part of the struct that implements the Getter interface, it can then call `getWithCredentials`.

Reply:
Done.